Makefile

```diff
@@ -1,10 +1,7 @@
 .PHONY: test
 test:
-	$(MAKE) generate -C lib/go
-	$(MAKE) test -C lib/go
```

Comment:
I don't want to get rid of the Go packages. Other repos depend on them, so we need to leave these in here in addition to the Cadence tests, same with ci

Reply:
Makes sense, I will bring them back, and fix the tests as well, with the updated imports :pray:

Reply:
Updated in https://github.com/onflow/flow-ft/pull/137/commits/4adf07d3d7c64e59116fd6ffc49ed9fb064bb806